frontend/src/scenes/insights/InsightDateFilter/InsightDateFilter.tsx

```diff
@@ -1,25 +1,29 @@
 import React from 'react'
-import { useValues, useActions } from 'kea'
+import { useActions, useValues } from 'kea'
 import { insightDateFilterLogic } from './insightDateFilterLogic'
 import { DateFilterProps, DateFilter } from 'lib/components/DateFilter/DateFilter'
 import './index.scss'
+import { insightLogic } from 'scenes/insights/insightLogic'
 
 export function InsightDateFilter(props: DateFilterProps): JSX.Element {
-    const logic = insightDateFilterLogic({ dateFrom: props.dateFrom, dateTo: props.dateTo })
+    const { insightProps } = useValues(insightLogic)
+    const { setFilters } = useActions(insightLogic)
+    const logic = insightDateFilterLogic({ dashboardItemId: undefined, ...insightProps })
```

Comment:
Why the extra `dashboardItemId: undefined` here?

Reply:
good catch, didn't mop up this leftover